group-coordinator/src/test/java/org/apache/kafka/coordinator/group/runtime/MultiThreadedEventProcessorTest.java

```diff
@@ -461,26 +459,30 @@ public void testMetrics() throws Exception {
     @Test
     public void testRecordThreadIdleRatioTwoThreads() throws Exception {
         GroupCoordinatorRuntimeMetrics mockRuntimeMetrics = mock(GroupCoordinatorRuntimeMetrics.class);
+        Time time = Time.SYSTEM;
```

Comment:
I used system time here because MockTime does not simulate on a per-thread basis